src/index.html

```diff
@@ -22,6 +22,288 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="./images/logo.png"
```

Comment:
Ensure that the image path is correct and the image exists at the specified location. This is more of a reminder to verify assets.